google-cloud-logging/lib/google/cloud/logging.rb

```diff
@@ -136,6 +136,14 @@ def self.new project_id: nil, credentials: nil, scope: nil, timeout: nil,
       # * `labels` - (Hash) User defined labels. A `Hash` of label names to
       #   string label values or callables/`Proc` which are functions of the
       #   Rack environment.
+      # * `set_default_logger_on_rails_init` - (Boolean) Whether it is safe for
```

Comment:
How about: "Whether Google Cloud Logging Logger should be allowed to start background threads and open gRPC connections on Rails initialization."

Reply:
Updated.